src/database/migrations/20191025142731-create-user.js

```diff
@@ -32,12 +32,12 @@ module.exports = {
       createdAt: {
         allowNull: false,
         type: Sequelize.DATE,
-        defaultValue: Sequelize.NOW
+        defaultValue: Sequelize.NOW,
       },
       updatedAt: {
         allowNull: false,
         type: Sequelize.DATE,
-        defaultValue: Sequelize.NOW
+        defaultValue: Sequelize.NOW,
```

Comment:
Same as well here:
field: 'updated_at',

Reply:
Ok,just avoiding the code not breaking again ni

Reply:
Please correct it